cupy/_core/core.pxd

```diff
@@ -105,8 +107,11 @@ cpdef Module compile_with_cache(str source, tuple options=*, arch=*,
 
 
 # TODO(niboshi): Move to _routines_creation.pyx
+
+cdef _zerocopy_fail()
+
```

Comment:
No need to add `_zerocopy_fail` to the header file because it is not called from external cython files.

```suggestion
```